Shelter.Java

```diff
@@ -0,0 +1,53 @@
+public class Shelter{
+/*The shelter class contains a var for the 
```

Comment:
While this comment isn't _wrong_ or anything, it could be made more useful. Describing what the class contains, while accurate, doesn't actually tell us what the class is used for, and we can figure out what fields it has just by reading the code. An improved comment might be something like "The Shelter class contains contact information for homeless shelters, and some validation logic for verifying phone numbers" or similar. Same applies for the user class.